API.md

```diff
@@ -63,115 +393,187 @@ This page documents the Party service API endpoints. All endpoints return an `HT
     "name": "Bolts and Ratchets Ltd"
 }
 ```
+<hr>
+
+#####Respondents Endpoints
+* `PUT /party-api/v1/respondents/email`
+* `PUT /party-api/v1/respondents/change_email`
+    * Changes a respondents old email address to a new provided one.
+        * Both the current email '`email_address`' and the '`new_email_address`' are to be provided in the body.
+    * Both URLs are linked to the same endpoint.
+    * This endpoint will update a respondent's email in the ras-party database and the oauth2 server. This email will need verified again so it will also set the user as unverified in the oauth server and will send a new verification email to the respondent.
```

Comment:
The email in the auth server is not updated by this end point but when the email is verified